src/control/cmd/daos_server/auto_test.go

```diff
@@ -479,6 +511,22 @@ func TestDaosServer_Auto_confGen(t *testing.T) {
 				return tc.hs, tc.hsErr
 			}
 
+			if tc.expOutPrefix != "" {
+				gotErr := cmd.confGenPrint(test.Context(t), gf, gs)
+				if gotErr != nil {
+					t.Fatal(gotErr)
+				}
+				if len(buf.String()) == 0 {
+					t.Fatal("no output from config generate print function")
+				}
+				outFirstLine := strings.Split(buf.String(), "\n")[0]
+				test.AssertTrue(t, strings.Contains(outFirstLine, tc.expOutPrefix),
```

Comment:
If it's a prefix, maybe `strings.HasPrefix` is a better way to check this?

Reply:
The text will have logging context before it including timestamp, we just want to ensure the first line contains the expected first-line of the config.

Reply:
changed to HasSuffix to provide a slightly more focused test